### ISSUE TITLE
Continue training on Flow matching: the loss downgrades but the generated audio is unintelligible and like noise.

### ISSUE THREAD

**Issue by huskyachao** [FunAudioLLM/CosyVoice#263]
Hi, when I tried to continue the training on the Conditional flow-matching on a new dataset (zh collected from youtube), I found that the loss degrade a lot but the generate audio is totally unintelligble (like noise) after around 3k steps. It seems that the network lost its ability to predict the vector field. Is there any suggestion about the training config or any other tricks about the continue training on the CFM (cosyvoice-300M)? Here is the main of my training config:
`train_conf:
    dist_backend: 'nccl'
    lr: 0.001
    scheduler: warmuplr
    scheduler_conf:
        warmup_steps: 20
    max_epoch: 10
    accum_grad: 2
    log_interval: 20
    grad_clip: 5
    save_per_step: 40
    batch_size: 2
`

**Comment by taalua** [comment]
I have similar problem fine-tuning the flow.

**Comment by ScottishFold007** [comment]
> Hi, when I tried to continue the training on the Conditional flow-matching on a new dataset (zh collected from youtube), I found that the loss degradee a lot but the generate audio is totally unintelligble (like noise) after around 3k steps. It seems that the network lost its ability to predict the vector field. Is there any suggestion about the training config or any other tricks about the continue training on the CFM (cosyvoice-300M)? Here is the main of my training config:
> 
> ```
> train_conf:
>     dist_backend: 'nccl'
>     lr: 0.001
>     scheduler: warmuplr
>     scheduler_conf:
>         warmup_steps: 20
>     max_epoch: 10
>     accum_grad: 2
>     log_interval: 20
>     grad_clip: 5
>     save_per_step: 40
>     batch_size: 2
> ```

How many hours in total? What are the training parameter settings? This information is needed to accurately pinpoint the issue.

**Comment by ScottishFold007** [comment]
maybe  the lr is too large~

**Comment by huskyachao** [comment]
> maybe the lr is too large~

Is there any suggested Lr for such a pretrained model? Or what is the suggested magnitude of the new Lr, e.g., 1e-6? Thanks.

**Comment by blackbird-fish** [comment]
I have the same question. I used about 200 hours data from scratch, but the auido still sounds noisy, and my lr is 1e-4, warmup step is 5000

**Comment by aluminumbox** [comment]
did you validated the code on libritts?  or the bug only appears on ZH data?

**Comment by blackbird-fish** [comment]
I just trained on ZH data

**Comment by bensonbs** [comment]
On demo dataset have same issues
